### ISSUE TITLE
bug - expectation error

### ISSUE THREAD

**Issue by Econ441** [h3rald/min#118]
Trying this:

```
( symbol bug
    (int :x string :y ==>)
    (x puts! y puts!)
) ::

?bug
1 2 bug
```

Results as:

```
=== bug [symbol]
(int :x string :y ==>)
(!) yolo.min(12,7) [bug]: Incorrect values found on the stack:
- expected: string int bug
- got:      int bug
    yolo.min(12,7) in symbol: bug
```

Problem is that parameter ordering of error is wrong:
Must be `expected: int string bug` not `expected: string int bug`

**Comment by h3rald** [comment]
Ok, so the expectation check works, but in case of an error the order is reversed, right?

**Comment by Econ441** [comment]
> Ok, so the expectation check works, but in case of an error the order is reversed, right?

Yep..

**Comment by h3rald** [comment]
Fixed in v0.31.0.
